File: periodoTres/POO/14-03/python/Pessoa.py
class Pessoa: # Classe
    def __init__(self, nome, idade): # Método para criar a classe
        self.nome = nome
        self.idade = idade
    
    def exibir_info(self): # Método
        print(f"Nome: {self.nome}")
        print(f"Idade: {self.idade}")

# Cria duas instâncias da classe Pessoa
pessoa1 = Pessoa("João", 30)
pessoa2 = Pessoa("Maria", 25)

# Chamando o método exibir_info() para cada instância
pessoa1.exibir_info()
print()
pessoa2.exibir_info()

File: periodoTres/POO/14-03/java/Pessoa/Pessoa.java
public class Pessoa {
    private String nome;
    private String sobrenome;
    private int idade;
    private double altura;
    private boolean trabalha;

    // Construtor da classe
    public Pessoa(String nome, String sobrenome,int idade, double altura, boolean trabalha) {
        this.nome = nome;
        this.sobrenome = sobrenome;
        this.idade = idade;
        this.altura = altura;
        this.trabalha = trabalha;
    }
    
    // Método para exibir informações da pessoa
    public void exibirInfo() {
        System.out.println("Nome: " + nome);
        System.out.println("Sobrenome: " + sobrenome);
        System.out.println("Idade: " + idade);
        System.out.println("Altura: " + altura);
        System.out.println("Está trabalhando: " + trabalha);
        System.out.println();
    }

    public static void main(String[] args) {
        // Criando duas instâncias da classe Pessoa
        Pessoa pessoa1 = new Pessoa("João", "Kleber", 30, 1.78, true);
        Pessoa pessoa2 = new Pessoa("Maria", "Clara", 25, 1.65, false);
    
        // Chamando o método exibirInfo() para cada instâncica
        pessoa1.exibirInfo();
        pessoa2.exibirInfo();
    }
}
